arm-machinelearning/2016-05-01-preview/swagger/commitmentPlans.json

```diff
@@ -0,0 +1,699 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure ML Commitment Plans Management Client",
+    "description": "These APIs allow end users to operate on Azure Machine Learning Commitment Plans resources and their child Commitment Association resources. They support the following operations:<ul><li>Create or update a commitment plan</li><li>Get a commitment plan</li><li>Patch a commitment plan</li><li>Delete a commitment plan</li><li>Get all commitment plans in a resource group</li><li>Get all commitment plans in a subscription</li><li>Get a commitment association</li><li>Get all commitment associations for a commitment plan</li><li>Move a commitment association from one commitment plan to another</li><li>Get usage history for a commitment plan</li></ul>",
```

Comment:
I think these html tags would be a problem in documentation of the generated client and they should be removed from the description.
@fearthecowboy - What do you think?